frontend/src/components/samples/SamplesListContent.js

```diff
@@ -115,6 +116,18 @@ const getTableColumns = (containersByID, individualsByID, projectsByID, sampleKi
       render: conc => conc !== null ? parseFloat(conc).toFixed(3) : null,
       width: 115,
     },
+    {
+      title: "QC Flag",
+      dataIndex: "qc_flag",
+      sorter: true,
+      render: (_, sample) => {
+        const flags = { quantity: sample.quantity_flag, quality: sample.quality_flag };
+        if (flags.quantity !== null && flags.quality !== null)
+          return <SampleQCFlag flags={flags}/>;
```

Comment:
The QC flag component should be usable for the library QC flags too. I would name it QCFlag simply for that reason.